### ISSUE TITLE
add process.thread.name to JUL formatter

### ISSUE THREAD

**Issue by pantovad** [elastic/ecs-logging-java#84]
Hello,

I wonder if you could add process.thread.name field to JUL formatter (tried through PR , but no perms):

```diff
diff --git a/jul-ecs-formatter/src/main/java/co/elastic/logging/jul/EcsFormatter.java b/jul-ecs-formatter/src/main/java/co/elastic/logging/jul/EcsFormatter.java index 940c5a5..30bda29 100644
--- a/jul-ecs-formatter/src/main/java/co/elastic/logging/jul/EcsFormatter.java
+++ b/jul-ecs-formatter/src/main/java/co/elastic/logging/jul/EcsFormatter.java
@@ -62,6 +62,7 @@ public class EcsFormatter extends Formatter {
         EcsJsonSerializer.serializeServiceName(builder, serviceName);
         EcsJsonSerializer.serializeEventDataset(builder, eventDataset);
         EcsJsonSerializer.serializeThreadId(builder, record.getThreadID());
\+        EcsJsonSerializer.serializeThreadName(builder, Thread.currentThread().getName().replaceAll("\\[", "_").replaceAll("]", "_"));
         EcsJsonSerializer.serializeLoggerName(builder, record.getLoggerName());
         if (includeOrigin && record.getSourceClassName() != null && record.getSourceMethodName() != null) {
             EcsJsonSerializer.serializeOrigin(builder, buildFileName(record.getSourceClassName()), record.getSourceMethodName(), -1);
```

It works nicely for me:

```json
{"@timestamp":"2020-07-31T14:43:23.827Z", "log.level": "INFO", "message":"HttpServerVerticle deployment complete.", "service.name":"myservice","event.dataset":"myservice.log","process.thread.id":71,"process.thread.name":"vert.x-eventloop-thread-0","log.logger":"myclass"}
```

**Comment by felixbarny** [comment]
That only works when not using an async appender that writes the logs in a different thread. Not sure how common that is for  JUL though.

I'm not aware that there are specific privileges needed for creating a PR.

**Comment by pantovad** [comment]
Raised PR from my fork and did the CLA ...

**Comment by felixbarny** [comment]
closed via #104
